www/web/tailwind.config.mjs

```diff
@@ -29,4 +29,5 @@ export default {
     },
   },
   plugins: [require('tailwind-scrollbar')],
+  darkMode: "class"
```

Comment:
```suggestion
  darkMode: "selector"
```
The selector strategy replaced the class strategy in Tailwind CSS v3.4.1.
src: https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually